SpotifyAPI/Web/SpotifyWebAPI.cs

```diff
@@ -44,6 +44,25 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+        #region Configuration
+        /// <summary>
+        /// Specifies after how many miliseconds should a failed request be retried.
+        /// </summary>
+        public int RetryAfter { get; set; } = 50;
+
+        /// <summary>
+        /// Should a failed request (Error 500, 502, or 503) be automatically retried or not.
+        /// </summary>
+        public bool UseAutoRetry { get; set; } = false;
+
+        /// <summary>
+        /// Maximum number of tries for one failed request.
+        /// </summary>
+        public int RetryTimes { get; set; } = 10;
+
+        public IEnumerable<int> RetryErrorCodes { get; private set; } = new int[] { 500, 502, 503 };
```

Comment:
@JohnnyCrazy  Hmm, I don't think I follow now. The only public properties/methods I've added are the configuration ones that are already "inline" documented. 

I've made no other changes to the public API of the library. I only changed some Utils method but non of them was documented before...

Or do you mean that I should just copy the "inline" docs to the proper documentation?